lib/ts/MemArena.cc

```diff
@@ -22,6 +22,8 @@
     limitations under the License.
  */
 
+#include <algorithm>
+
```

Comment:
What's this for?

Reply:
`std::max` needs it and it was being included by `string_view.h` before.